msom/src/main/java/pl/edu/uj/fais/wpz/msom/entities/Task.java

```diff
@@ -21,7 +22,7 @@
 
     private String name;
 
-    @ManyToOne
+    @ManyToOne(cascade = {CascadeType.ALL})
```

Comment:
Przypadkiem to chyba wrzuciłem jak testowałem usuwanie. Wystarczy usunąć to kaskadowanie (defaultowo nie kaskaduje nic).